docs/src/main/asciidoc/update-to-quarkus-3.adoc

```diff
@@ -0,0 +1,55 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="update-projects-to-quarkus-3-automatically-howto"]
+= Update projects to Quarkus 3 automatically
+include::_attributes.adoc[]
+:categories: core
+:extension-status: "experimental"
+:summary: Update projects from Quarkus 2.13 and later to Quarkus 3.
+
+include::{includes}/extension-status.adoc[]
+
+As a developer, you can update your projects from Quarkus 2.13 and later to Quarkus 3 by running an update command.
+
+The update command uses OpenRewrite recipes to automatically update most of your project's dependencies, source code, and documentation. These recipes cover many but not all of the items described in the link:https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0[Migration Guide 3.0].
+
+After updating the project, if you do not find all the updates you expect, there are two possible reasons:
+- The recipe might not cover an item in your project.
+- Your project might use an extension that does not support Quarkus 3 yet.
+
+In either case, https://github.com/quarkusio/quarkus/issues[let us know by filing an issue] so we can improve the update recipe.
```

Comment:
```suggestion
In either case, https://github.com/quarkusio/quarkus/issues[let us know by filing an issue] so we can improve the update command.
```

its not just recipes that can be at fault; also the auxiliary code/commands used to implement it.

Reply:
Updated. Thanks.